sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlockBlobApiTests.java

```diff
@@ -1115,8 +1116,7 @@ private static Stream<Arguments> downloadFileSupplier() {
             Arguments.of(50 * Constants.MB, EncryptionVersion.V1), // large file requiring multiple requests
             Arguments.of(0, EncryptionVersion.V2), // empty file
             Arguments.of(20, EncryptionVersion.V2), // small file
-            //Arguments.of(16 * 1024 * 1024, EncryptionVersion.V2), // medium file in several chunks
-            //todo isbr: uncomment after 4mb boundary bug is fixed
+            Arguments.of(16 * 1024 * 1024, EncryptionVersion.V2), // medium file in several chunks
```

Comment:
I think there may be a couple more tests that we'll need to uncomment like this one: https://github.com/Azure/azure-sdk-for-java/blame/27bef9ead66da9221f481063654c32829e189f99/sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlockBlobApiTests.java#L1895. It may make sense to sync up with @ibrahimrabab to make sure we uncomment all of the test cases that we needed to comment out because of this issue.